spec/descriptions/resumeMaintenanceConfig.md

```diff
@@ -0,0 +1,7 @@
+This endpoint resumes a maintenance configuration given its ID.
+
+### Path Parameters:
+
+- **id:** The ID of the maintenance config to resume.
+
+_Note: When you resume a maintenance configuration, you will not receive alerts for the scope defined in the maintenance configuration._
```

Comment:
The note here is not correct. Once fixed, can we just move it in the same paragraph as `This endpoint resumes a maintenance configuration given its ID.` ?